js/coinex.js

```diff
@@ -3474,9 +3471,18 @@ module.exports = class coinex extends Exchange {
         const request = {
             'amount': amountToPrecision,
             'coin_type': currency,
-            'transfer_side': transfer, // 'in': spot to swap, 'out': swap to spot
         };
-        const response = await this.privatePostContractBalanceTransfer (this.extend (request, query));
+        if ((marketType === 'spot') || (marketType === 'swap')) {
```

Comment:
In this case, the `transfer` method should not really depend on a single marketType setting, cause it transfers from one market type to another. So it is not semantically correct to use the default market type setting to drive the `transfer` logic. I suggest that we just detect if either `fromAccount` or `toAccount` is not `'swap'` nor `'spot'` and then it's a transfer between margin accounts.